sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -309,6 +340,9 @@ class RapidsDriverPlugin extends DriverPlugin with Logging {
     val sparkConf = pluginContext.conf
     RapidsPluginUtils.fixupConfigsOnDriver(sparkConf)
     val conf = new RapidsConf(sparkConf)
+    if (!conf.allowMultipleJars) {
```

Comment:
I think we should still detect it even with this setting, just change error it to a warning instead of throwing an exception. you can move this check into detectMultiplePluginJars

Reply:
done.